### PR TITLE
HTML/CSS Project - YoutubeSite

### PR DESCRIPTION
**Pull request by Milypm**
**Description:**
Project created with HTML5 and CSS3 based on the main Youtube site, including a Navbar, a video section, and a suggested video descriptive section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+    <body>
+
+        <div class="main-cont">
+
+            <nav class="nav-cont">
+
+                <div class="logo-cont">
+                    <div class="options-icon">
+                        <img src="images/options-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an **alt** attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                    </div>
+                    <div>
+                        <img src="images/new-logo.jpg" class="yt-logo">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+                <div class="search-cont">
+                    <form class="search-btn">
+                        <input type="text" placeholder="Search..">
+                        <button type="submit"><i class="fa fa-search"></i></button>
+                    </form>
+                </div>
+    
+                <div class="icons-cont">
+                    <div class="video-icon">
+                        <img src="images/video-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                    </div>
+                    <div class="apps-icon">
+                        <img src="images/apps-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+                    </div>
+                    <div class= "bell-icon">
+                        <img src="images/bell-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+                    </div>
+                    <div class="user-icon">
+                        <img src="images/user-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+                    </div>
+                </div>
+     
+            </nav>
+    
+<!--------------------------- MAIN VIEW: Video - Description - videos/right-------->
```
Comment (arslanbisharat) [inline index.html]: The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+            <div class = "mainview-cont">
+
+                <section class = "left-section">
+
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/JhPC6_rO08s" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (arslanbisharat) [inline index.html]: The frameborder attribute on the iframe element is obsolete

File: index.html [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+                
+                    <section class="videoDescrip-section">
+                        
+                        <h3 class= "video-font">Contributing to Open Source Part I: The Easy Way</h3>
+                        <section class="subscribers-section">
+                            <div class="avatar-cont">
+                                <a href="https://www.youtube.com/watch?v=M_pZReasTvE"><img src="images/t-avatar.png"></a>
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x] make more commits

- [x] Please make it correctly align as present on the original youtube page.
chrome-capture (6)

- [ ]  There should no gap between heading and video.
chrome-capture (7)
 

- [x] Please make you readme more interesting by adding headings and other formats. You can also find a sample of good Readme files from your project requirements. 

- [x] An img element must have an alt attribute

- [x] Your PR is not descriptive. Please add a message to your PR otherwise your project will invalidate. 



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by arslanbisharat** [comment]
Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

File: README.md [hunk 10]
```diff
@@ -1,2 +1,65 @@
 # 1st-Project-YoutubeSite
-1st HTML/CSS project.
+# Embedded HTML/CSS project. :grinning: :+1:
+
+:EMOJICODE:
```
Comment (teekaytech) [inline README.md]: Please, remove this line from your readme

File: README.md [hunk 11]
```diff
@@ -1,2 +1,65 @@
 # 1st-Project-YoutubeSite
-1st HTML/CSS project.
+# Embedded HTML/CSS project. :grinning: :+1:
```
Comment (teekaytech) [inline README.md]: Emojis are not necessary for project title. Always keep your readme professional as much as you can. Kindly remove the emojis.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

### Hello @Milypm  :wave: 

**You've done an amazing job so far. However, there are still some changes to be made to have a complete project. Let's quickly talk about it so you can make the changes. Kindly submit another review request when you're done.**


**[REQUIRED]**
_Compulsory changes to be made_
- The previous TSE mentioned something about making your readme descriptive. You've really done justice to that and it's commendable. Kindly review my points below and fix your readme file.

**[OPTIONAL]**
_Not compulsory, although will make a nice feature in your project_
- You're advised to implement the footer and comments sections of the page.

Kindly fix the changes and submit another review request form.

**I'm @taofeekolalere on slack.
Happy Coding:man_technologist:**

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

**Comment(s)**
- In your next project, you should let your pull request message be more descriptive.
- Also, let your commit messages be more descriptive. [Check this article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8) to know more.

Again, congratulations, you can merge to master.
Good luck with your next project.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**